Makefile

```diff
@@ -168,15 +168,15 @@ generate.init: $(TERRAFORM_PROVIDER_SCHEMA) pull-docs
 # Test utilities
 # TODO(muvaf): Move most of this to build submodule.
 
-uptest: $(KIND) $(KUBECTL) $(HELM3) $(UP) $(KUTTL)
+uptest: $(KIND) $(KUBECTL) $(HELM3) $(UP) $(KUTTL) $(UPTEST)
```

Comment:
@muvaf I am seeing `uptest` not installed at all:

<img width="404" alt="Screen Shot 2022-10-17 at 12 57 10" src="https://user-images.githubusercontent.com/9900707/196218543-bf77e689-2aec-4f1c-bcde-dd74f9885d28.png">

I suspect this could be related to [these steps](https://github.com/upbound/provider-aws/blob/main/.github/workflows/uptest.yml#L84-L96) in the CI workflow where code is checked out first with submodules (i.e. latest main) and PR checked out afterwards requiring a `git submodule update --init` to get latest build submodule. 

Here is what I see in my local:

```shell
❯ git reset --hard upstream/main
HEAD is now at ca5cf438 readme: fix line break
❯ git co use-uptest
M	build
Switched to branch 'use-uptest'
Your branch is up to date with 'muvaf/use-uptest'.
❯ git status
On branch use-uptest
Your branch is up to date with 'muvaf/use-uptest'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   build (new commits)

no changes added to commit (use "git add" and/or "git commit -a")
```

So, I suspect the following PR could fix the problem: https://github.com/upbound/provider-aws/pull/84

Reply:
Thank you @turkenh !!